swagger-initializer.js

```diff
@@ -1,14 +1,15 @@
 window.onload = function() {
   //<editor-fold desc="Changeable Configuration Block">
 
-  // the following lines will be replaced by docker/configurator, when it runs in a docker-container
+  // Changes from defaults include the url and skipping the first standalone
+  // preset, as that loads the TopBar which has XSS problems.
   window.ui = SwaggerUIBundle({
     url: "https://zipkin.io/zipkin-api/zipkin2-api.yaml",
     dom_id: '#swagger-ui',
     deepLinking: true,
     presets: [
       SwaggerUIBundle.presets.apis,
-      SwaggerUIStandalonePreset
+      SwaggerUIStandalonePreset.slice(1) // index zero is the TopBar
```

Comment:
Maybe worth linking the file since I guess this is reflecting implementation detail

https://github.com/swagger-api/swagger-ui/blob/master/src%2Fstandalone%2Fpresets%2Fstandalone%2Findex.js

Reply:
done. that's a very good idea.